### ISSUE TITLE
How the weights of different loss functions affect performance of the network in GAN based SISR?

### ISSUE THREAD

**Issue by KhushbooChauddhary** [tensorlayer/srgan#245]
Total perceptual loss in SRGAN paper is weighted sum of content loss and adversarial loss.

Total loss = Content loss + (10^(-3)) Adversarial loss Please tell why 10^(-3) is used? what is its impact on performance if some other value is used ?? or does it affects number of iterations for training of network ?

**Comment by hanjr92** [comment]
![20190501173641590](https://user-images.githubusercontent.com/73918561/177238381-561fed18-fb3d-4db0-9208-60ba57e8928e.png)
sorry too late to reply you. In paper, we can find the total loss = content loss + (10^(-3)) Adversarial loss.
